metadata/src/main/java/org/apache/kafka/controller/FeatureControlManager.java

```diff
@@ -316,10 +326,12 @@ public boolean hasNext() {
         public List<ApiMessageAndVersion> next() {
             // Write the metadata.version first
             if (!wroteVersion) {
-                wroteVersion = true;
-                return Collections.singletonList(new ApiMessageAndVersion(new FeatureLevelRecord()
-                    .setName(MetadataVersion.FEATURE_NAME)
-                    .setFeatureLevel(metadataVersion.featureLevel()), FEATURE_LEVEL_RECORD.lowestSupportedVersion()));
+                if (!metadataVersion.isLessThan(minimumBootstrapVersion)) {
```

Comment:
nit: i think this check reads easier as `metadataVersion.isAtLeast(minimumBootstrapVersion)`